.github/workflows/publish.yml

```diff
@@ -0,0 +1,73 @@
+name: Deploy Next.js site to Pages
+
+on:
+  # Runs on pushes targeting the default branch
+  push:
```

Comment:
Remove this, there won't be any website on this repo, so it is only running from other repos.